boot/zephyr/CMakeLists.txt

```diff
@@ -210,9 +210,13 @@ if(CONFIG_MCUBOOT_SERIAL)
     )
 endif()
 
+get_filename_component(CONF_DIR ${CONF_FILE} DIRECTORY)
```

Comment:
Since `CONF_FILE` is not used elsewhere in MCUBoot repository, you might want to add a comment stating that `CONF_FILE` points to the kconfig configuration file (e.g. prj.conf)

Reply:
ok